docs/en/observability/deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,296 @@
+:aws: AWS
+
+[[deploy-elastic-serverless-forwarder]]
+= Deploy Elastic Serverless Forwarder
+
+++++
+<titleabbrev>Deploy</titleabbrev>
+++++
+:keywords: serverless
+:description: Deploy the Elastic Serverless Forwarder using Kibana and the AWS Serverless Application Repository (SAR).
+
+To deploy Elastic Serverless Forwarder, you have to:
+
+* <<aws-serverless-forwarder-deploy-kibana>>
+* <<sample-s3-config-file>>
+* <<aws-serverless-forwarder-deploy-sar>>
+
+NOTE: This page describes the basic steps required to deploy Elastic Serverless Forwarder — for additional detail on configuration topics such as permissions and automatic routing, see <<configure-elastic-serverless-forwarder>>; for additional detail on parsing and enriching data, see <<aws-serverless-transform>>.
+
+[[aws-serverless-forwarder-deploy-kibana]]
+== Install {aws} integration assets in {kib}
+
+. Go to **Integrations** in {kib} and search for {aws} (or select the **{aws}** category to filter the list).
+. Click the {aws} integration, select **Settings** and click **Install {aws} assets** and confirm to install all the {aws} integration assets.
+
+[role="screenshot"]
+image::images/aws-serverless-forwarder-install-assets.png[Find and install AWS integration assets in {kib}]
+
+Adding integrations from {kib} provides appropriate pre-built dashboards, ingest node configurations, and other assets that help you get the most out of the data you ingest. The integrations use https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html[data streams] with specific https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme[naming conventions] that provide you with more granular controls and flexibility on managing data ingestion.
+
```

Comment:
Let's add a Note here. Integration assets are our preferred way but we support writing to any index or data stream.
Note: While Elastic recommends using the integration assets, the Lambda function also supports writing directly to an index, alias, or custom data stream. This enables existing Elasticsearch users to re-use index templates, ingest pipelines, or dashboards that are already created and connected to business processes. If you already have an index or data stream set up where you want to send the data, then you can skip the above step.

Reply:
Added in next commit...